### ISSUE TITLE
Update copyright year to 2015

### ISSUE THREAD

**Issue by ThiefMaster** [indico/indico#1709]
*It's probably a good idea to do this this* ***after the new payment module is merged into master.*** *Otherwise we end up with the same chaos as last year (big changes being integrated with old dates in the header).*

Also don't forget the plugin repo! All files there should have the proper style and just need the year to be updated!

Maybe we can take this as a good opportunity to use the same header style everywhere (some files still use 2013, some use `##` vs `#`, some have the unnecessary utf8 header, some have a missing space after a semicolon in the GPL notice). Right now you need to be careful from which file you copy the header because of the differences.

My suggestion would be to use this one, which is the one which we already use in all files added in the last months (besides the date, obviously):

```python
# This file is part of Indico.
# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
#
# Indico is free software; you can redistribute it and/or
# modify it under the terms of the GNU General Public License as
# published by the Free Software Foundation; either version 3 of the
# License, or (at your option) any later version.
#
# Indico is distributed in the hope that it will be useful, but
# WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
# General Public License for more details.
#
# You should have received a copy of the GNU General Public License
# along with Indico; if not, see <http://www.gnu.org/licenses/>.
```

Same one for JS, just with JS-style comments:

```javascript
/* This file is part of Indico.
 * Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
 *
 * Indico is free software; you can redistribute it and/or
 * modify it under the terms of the GNU General Public License as
 * published by the Free Software Foundation; either version 3 of the
 * License, or (at your option) any later version.
 *
 * Indico is distributed in the hope that it will be useful, but
 * WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 * General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with Indico; if not, see <http://www.gnu.org/licenses/>.
 */
```

**Comment by ThiefMaster** [comment]
After running the script a few files need to be fixed manually because someone pasted nonbreaking spaces and similar non-ascii chars which never got noticed because of the UTF8 header.

**Comment by ThiefMaster** [comment]
Closed in 241922c
